babel.config.js

```diff
@@ -159,6 +159,7 @@ module.exports = function (api) {
       convertESM ? "@babel/proposal-export-namespace-from" : null,
       convertESM ? "@babel/transform-modules-commonjs" : null,
       convertESM ? pluginNodeImportInteropBabel : pluginNodeImportInteropRollup,
+      unshadowGlobalCjsRequire,
```

Comment:
We should also remove `import { createRequire } from "module"` otherwise rollup will complain [`createRequire` is not found](https://github.com/babel/babel/runs/2228352592#step:6:567).